### PR TITLE
sorts for Panel Meetings and removing unused textInput

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/753)

### CODE REVIEW

File: src/Constants/Sort.js [hunk 1]
```diff
@@ -223,8 +223,6 @@ AGENDA_EMPLOYEES_SORT.defaultSort = AGENDA_EMPLOYEES_SORT.options[0].value;
 
 export const PANEL_MEETINGS_SORT = {
   options: [
-    { value: 'meeting_date', text: 'Meeting Date Asc.' },
```
Comment (SidelineCory24) [inline src/Constants/Sort.js:226]: 🧼

**Comment by SidelineCory24** [comment]
PR Review Checklist
- `textSearch/id search` still working as expected after removing `textInput`
- Sort by only shows meeting status a-z/z-a

**Comment by SidelineCory24** [comment]
@elizabeth-jimenez are you able to get two or more statuses to return? I can only see one at a time so I can't verify on the site itself if the sort by is working. I do see the network request updating properly with the correct sort option.

image

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez are you able to get two or more statuses to return? I can only see one at a time so I can't verify on the site itself if the sort by is working. I do see the network request updating properly with the correct sort option.
> 
> image

That's not a part of this ticket - but was addressed as a fix [PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/749) for my previous Panel Filters one

**Comment by SidelineCory24** [comment]
> > @elizabeth-jimenez are you able to get two or more statuses to return? I can only see one at a time so I can't verify on the site itself if the sort by is working. I do see the network request updating properly with the correct sort option.
> > image
> 
> That's not a part of this ticket - but was addressed as a fix [PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/749) for my previous Panel Filters one

Ah okay. Should we expect it to work then in our local? I double checked my branches were up to date and I am still seeing no results.

**Comment by SidelineCory24** [comment]
> > @elizabeth-jimenez are you able to get two or more statuses to return? I can only see one at a time so I can't verify on the site itself if the sort by is working. I do see the network request updating properly with the correct sort option.
> > image
> 
> That's not a part of this ticket - but was addressed as a fix [PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/749) for my previous Panel Filters one - Or u may be talking about our Mock not being up to date for sorting/filtering, which is common

right the mock, understood

**Comment by SidelineCory24** [comment]
@elizabeth-jimenez Is it okay to merge dev into this branch? Don't want to mess up anything you may be working on

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez Is it okay to merge dev into this branch? Don't want to mess up anything you may be working on

Yep! Thanks for asking!

File: src/Constants/Sort.js (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/Constants/Sort.js:226]: ?

Comment (SidelineCory24) [reply src/Constants/Sort.js:226]: not action needed, just referencing that you deleted it, bad emoji usage🙃

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go!🚀

**Pull request merged**